crates/store/src/db/tests.rs

```diff
@@ -449,6 +450,17 @@ fn sql_unconsumed_network_notes() {
     )
     .unwrap();
     assert_eq!(result, network_notes);
+    let (result, _) = sql::unconsumed_network_notes_for_network_account(
+        &db_tx,
+        network_account_id.try_into().unwrap(),
+        block_num,
+        Page {
+            token: None,
+            size: NonZeroUsize::new(N as usize * 10).unwrap(),
+        },
+    )
+    .unwrap();
+    assert_eq!(result, network_notes);
```

Comment:
This is probably an offensive way to test the new functionality (by just pasting it into the existing test for the similar existing fn).

LMK if you have any ideas how better to test this.

Reply:
@Mirko-von-Leipzig

Reply:
I definitely don't like it; but it is on par with the rest of our store tests so I think its okay (for now).

What I would love is overhauling the store tests to be more genuine. Using `MockChain` to generate realistic blocks, and then applying them to the `store`. And then running tests on that.